src/engine/source/router/src/orchestrator.cpp

```diff
@@ -209,7 +210,7 @@ base::OptError Orchestrator::initWorker(const std::shared_ptr<IWorker>& worker,
 // Public
 void Orchestrator::Options::validate() const
 {
-    if (m_numThreads < 1 || m_numThreads > 128)
+    if (m_numThreads < 0 || m_numThreads > 128)
     {
         throw std::runtime_error {"Configuration error: numThreads must be between 1 and 128"};
```

Comment:
fix comment: 
`throw std::runtime_error {"Configuration error: numThreads must be between 0 and 128"};`